Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -550,25 +550,29 @@ public Maybe<OperationError> UpdateSystemVersion(string version)
 
         public void ResetUserCount()
         {
-            UserCount = UserCount.BELOWTEN;
+            UserCount = null;
         }
 
-        public Maybe<OperationError> SetExpectedUsersInterval((int lower, int? upperBound) newIntervalValue)
+        public Maybe<OperationError> SetExpectedUsersInterval((int? lower, int? upperBound) newIntervalValue)
         {
             switch (newIntervalValue)
             {
+                case (null, 100):
```

Comment:
wierd fallback for null,100
Will be fixed once Maybe is leveraged in stead to reset the field to null.